src/templates/smfcfg.yaml.j2

```diff
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: 2022-present Intel Corporation
+# SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
+# Copyright 2019 free5GC.org
+#
+# SPDX-License-Identifier: Apache-2.0
+
+info:
+  version: 1.0.0
+  description: SMF initial local configuration
+
+configuration:
+  smfDBName: {{ smf_database_name }}
+  enableDBStore: false
+  enableUPFAdapter: true
+  debugProfilePort: 5001
+  mongodb:
+    name: {{ default_database_name }}
+    url: {{ default_database_url }}
+  kafkaInfo:
+    brokerUri: "sd-core-kafka-headless"
+    brokerPort: 9092
+    topicName: "sdcore-data-source-smf"
+  smfName: SMF # the name of this SMF
+  sbi: # Service-based interface information
+    scheme: http # the protocol for sbi (http or https)
+    registerIPv4: {{ smf_url }} # IP used to register to NRF
+    bindingIPv4: smf # IP used to bind the service
+    port: {{ smf_sbi_port }} # Port used to bind the service
+    tls: # the local path of TLS key
+      key: free5gc/support/TLS/smf.key # SMF TLS Certificate
+      pem: free5gc/support/TLS/smf.pem # SMF TLS Private key
+  serviceNameList: # the SBI services provided by this SMF, refer to TS 29.502
+    - nsmf-pdusession # Nsmf_PDUSession service
+    - nsmf-event-exposure # Nsmf_EventExposure service
+    - nsmf-oam # OAM service
+  pfcp: # the IP address of N4 interface on this SMF (PFCP)
+    addr: {{ pod_ip }}
+  userplane_information: # list of userplane information
+    up_nodes: # information of userplane node (AN or UPF)
+      gNB: # the name of the node
+        type: AN # the type of the node (AN or UPF)
+      UPF1: # the name of the node
```

Comment:
I'm pretty sure everything here would be delivered through the webui/simapp. The `sst` and `sd` are part of what's provided [there](https://github.com/gruyaume/omec-simapp-operator/blob/main/src/files/default_config.yaml). If possible, remove references this whole block.

Reply:
Removed as discussed.